rts/lra/API.adoc

```diff
@@ -0,0 +1,51 @@
+= Versioning of Narayna LRA REST API
+
+The goal of this document is summarize the approach to REST API versioning
+in Narayana LRA coordinator service.
+This document is written for developer of Narayana LRA services.
+
+The version format is `major.minor`[`-preRelease`].
+The `major` and `minor` parts are required and `-preRelease` is used
+during development and is optional.
+
+NOTE: Any final `minor.major` release is considered higher than
+      the same version with `-preRelease` part.
+
+Client may demand behaviour based on particular API version
+by providing HTTP header link:./service-base/src/main/java/io/narayana/lra/LRAConstants.java[`Narayana-LRA-API-version`] on the call.
```

Comment:
You make the assumption that the value in the version header must exactly match the supported version, ie white space is significant. I guess that requirement is implicit?